src/plugins/intellisense/state.tsx

```diff
@@ -0,0 +1,208 @@
+import { BoundIdentifier } from ".";
+import { mapAugAST } from "./latex-parsing";
+import { ItemModel } from "globals/models";
+import { Calc } from "globals/window";
+import { rootKeys } from "plugins/find-replace/backend";
+import Metadata from "plugins/manage-metadata/interface";
+import { parseRootLatex } from "plugins/text-mode/aug/rawToAug";
+
+export function get<T extends object, K extends string | symbol | number>(
+  t: T,
+  prop: K
+): K extends keyof T ? T[K] : undefined {
+  // eslint-disable-next-line @typescript-eslint/consistent-indexed-object-style
+  const obj2 = t as {
+    [Key in string | symbol | number]: Key extends keyof T ? T[Key] : undefined;
+  };
+
+  return obj2[prop];
+}
```

Comment:
This feels more appropriate for a typescript-helpers file. Since we don't have one, src/utils/utils.ts can do.

Reply:
Fixed!